tasks/sysctl-setup.yml

```diff
@@ -7,7 +7,24 @@
     ansible_distribution != 'Debian'
     or ansible_distribution_major_version | int < 10
 
-# See: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#letting-iptables-see-bridged-traffic
```

Comment:
This link routed to a now seemingly gone part of the article. I think the new link which I've included covers what this one would have.